lib/usbd/backend/usbd_dwc_otg.c

```diff
@@ -919,6 +921,18 @@ static void process_out_endpoint_interrupt(usbd_device *dev, uint8_t ep_num)
 		LOGF_LN("Transfer Complete: endpoint 0x%"PRIx8, ep_addr);
 		usbd_urb *urb = usbd_find_active_urb(dev, ep_addr);
 
+		if (is_ep_prematured(dev, ep_addr)) {
+			if (urb != NULL){
+				//* ommit this RXC and restart current URB
```

Comment:
C style comments less then 5lines - disturbing me, especialy if it goes in a distance of 1-10lines of code